packages/discovery/Discovery.yml

```diff
@@ -92,6 +92,15 @@ apis:
         apiType: openapi-v3
         tags:
           - integrations-and-notifications
+      - id: launch
```

Comment:
This one is missing the `url` property.

Reply:
🤦 I'll get that fixed in a little bit, thanks for catching that.

Reply:
@josejulio done